src/main/java/net/snowflake/client/config/SFClientConfigParser.java

```diff
@@ -50,14 +50,14 @@ public static SFClientConfig loadSFClientConfig(String configFilePath) throws IO
             Paths.get(systemGetProperty("user.home"), SF_CLIENT_CONFIG_FILE_NAME).toString();
         if (Files.exists(Paths.get(userHomeFilePath))) {
           derivedConfigFilePath = userHomeFilePath;
-        } else {
```

Comment:
why only commenting it out instead of removing it entirely or enable/disable it with a feature flag?